test/datasets/test_qnli.py

```diff
@@ -54,7 +54,7 @@ class TestQNLI(TempDirMixin, TorchtextTestCase):
     def setUpClass(cls):
         super().setUpClass()
         cls.root_dir = cls.get_base_temp_dir()
-        cls.samples = _get_mock_dataset(cls.root_dir)
+        cls.samples = _get_mock_dataset(os.path.join(cls.root_dir, "datasets"))
```

Comment:
Can we remove this change from this PR. I will make all fixes corresponding to QNLI dataset in this PR https://github.com/pytorch/text/pull/1760